specification/docs/slices.md

```diff
@@ -17,101 +20,110 @@ All slices produce machine-readable json output that can be parsed using atom [p
-            "label" : "LOCAL"
+          "targetObj": {
+            "name": "username",
+            "typeFullName": "java.lang.String",
+            "lineNumber": 276,
+            "columnNumber": 3,
+            "label": "LOCAL"
           },
 
 ```
 
-3. Now iterate the `usages` array. Under the each category the attributes `typeFullName` (Found in `targetObj` and `definedBy`) and `resolvedMethod` (`invokedCalls` and `argToCalls`) are interesting.
+3. Iterate over the `usages` array. The attributes `typeFullName` (found in `targetObj`
+   and `definedBy`) and `resolvedMethod` (`invokedCalls` and `argToCalls`) under each category are of interest.
 
-4. Now iterate the `userDefinedTypes` array. For each type, note down the `fileName` and `lineNumber`. For each `field`, the attribute `typeFullName` indicates the aliased field. For each `procedure`, the `paramTypes` array would list the custom type from index 1 onwards.
+4. Iterate over the `userDefinedTypes` array. Note the `fileName` and `lineNumber` for each type. For each `field`,
+   the attribute `typeFullName` indicates the aliased field. For each `procedure`, the `paramTypes` array lists the
+   custom type from index 1 onward.
 
 ```json
 "userDefinedTypes" : [
 {
-    "name" : "com.example.vulnspring.WebController",
-    "fields" : [
-    {
-        "name" : "jdbcTemplate",
-        "typeFullName" : "org.springframework.jdbc.core.JdbcTemplate",
-        "lineNumber" : 42,
-        "columnNumber" : 15,
-        "label" : "LOCAL"
-    },
-    {
-        "name" : "logger",
-        "typeFullName" : "org.slf4j.Logger",
-        "lineNumber" : 44,
-        "columnNumber" : 30,
-        "label" : "LOCAL"
-    }
-    ],
-    "procedures" : [
-    {
-        "callName" : "home",
-        "resolvedMethod" : "com.example.vulnspring.WebController.home:java.lang.String(org.springframework.ui.Model,javax.servlet.http.HttpSession)",
-        "paramTypes" : [
-        "com.example.vulnspring.WebController",
-        "org.springframework.ui.Model",
-        "javax.servlet.http.HttpSession"
-        ],
-        "returnType" : "java.lang.String",
-        "lineNumber" : 46,
-        "columnNumber" : 2
-    },
+"name": "com.example.vulnspring.WebController",
+"fields": [
+{
+"name": "jdbcTemplate",
+"typeFullName": "org.springframework.jdbc.core.JdbcTemplate",
+"lineNumber" : 42,
+"columnNumber": 15,
+"label": "LOCAL"
+},
+{
+"name": "logger",
+"typeFullName": "org.slf4j.Logger",
+"lineNumber": 44,
+"columnNumber": 30,
+"label" : "LOCAL"
+}
+],
+"procedures": [
+{
+"callName": "home",
+"resolvedMethod": "com.example.vulnspring.WebController.home:java.lang.String(org.springframework.ui.Model,javax.servlet.http.HttpSession)",
+"paramTypes" : [
+"com.example.vulnspring.WebController",
+"org.springframework.ui.Model",
+"javax.servlet.http.HttpSession"
+],
+"returnType": "java.lang.String",
+"lineNumber": 46,
+"columnNumber": 2
+},
 ```
 
-### Data Flow slice
+### Data flow slice
```

Comment:
Let's use capital Flow

Reply:
@prabhu If I capitalize flow in this case, it creates an inconsistency for either every other reference to data flows or the formatting of headings. Capitalizing Data Flow but not slices means that data flow is a proper noun and should always be capitalized everywhere - and Usages as well. If I also capitalize slices to avoid this, that means the formatting will be inconsistent with the format of every other heading. I can do either one, or ignore the inconsistency - please just confirm which outcome is the desired one.

Reply:
Ah, good point. Lets use your version with lower case flow